### PR TITLE
Apps 5658 secure=true by default

### PR DESCRIPTION
**Pull request by yaniv-li**

### CODE REVIEW

File: src/video-player.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     const initCloudinary = () => {
       const opts = _options.cloudinary;
       opts.chainTarget = this;
-
+      _options.cloudinary.cloudinaryConfig.config('secure', true);
```
Comment (maoznir) [inline src/video-player.js:315]: Adding this to defaults.js would be better no?

**Review by maoznir — CHANGES_REQUESTED** [review]

File: src/video-player.js (hunk 1 above)
Comment (yaniv-li) [reply src/video-player.js:315]: ok

File: test/unit/cloudinaryConfig.test.js.mv [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const cld = cloudinary.Cloudinary.new({ cloud_name: 'demo' });
+cloudinary.VideoPlayer = VideoPlayer;
+
+
+describe('secure true test', () => {
+  it('test force secure true', () => {
+    jest.useFakeTimers();
+    document.body.innerHTML = '<div><video id="test"/></div>';
+    let vp = new VideoPlayer('test', { hideContextMenu: true }, false);
+    console.log(vp);
```
Comment (maoznir) [inline test/unit/cloudinaryConfig.test.js.mv:15]: @yaniv-li can you please add a test that checks the source is https
and a test that checks that when passing secure=false to cloudinary, it is http?

**Review by maoznir — CHANGES_REQUESTED** [review]

File: test/unit/cloudinaryConfig.test.js.mv (hunk 2 above)
Comment (yaniv-li) [reply test/unit/cloudinaryConfig.test.js.mv]: @maoznir I added explicit secure true **in the player options** not in the cloudinary options  
there are 2 tests for default  and explicit false

**Review by maoznir — APPROVED** [review]

**Review by maoznir — CHANGES_REQUESTED** [review]
cloudinaryConfig.test.js fails

**Pull request merged**